unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendsTabView.cs

```diff
@@ -70,4 +96,39 @@ protected override void OnPressDeleteButton(FriendEntryBase entry)
             OnDeleteConfirmation?.Invoke(entry as FriendEntry);
         });
     }
+
+    private void ChatController_OnAddMessage(ChatMessage message)
+    {
+        if (message.messageType == ChatMessage.Type.PRIVATE)
```

Comment:
We can invert this if so the indenting is reduced.

Reply:
Done!